sitemap/templates/career-email-template.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
I think We do not need this, because jobs repository will use its own mail template, it will not use sitemap's template